Mage.Client/src/main/java/mage/client/table/TablesPanel.java

```diff
@@ -756,8 +756,8 @@ public void showTables(UUID roomId) {
         this.roomId = roomId;
         UUID chatRoomId = null;
         if (SessionHandler.getSession() != null) {
-            btnQuickStartDuel.setVisible(SessionHandler.isTestMode());
```

Comment:
GUI elements must be changed by NetBeans 8 in designer mode (current form is openable, so it must be changed), so that changes will be saved in `*.form` file too. Changes in java only file will be lost after any GUI edits.

P.S. I think renames can be made without real GUI edit -- just open `*.form` file, find and replace new names.

Reply:
The download link is broken: https://netbeans.apache.org/download/archive/index.html

Can this form be upgrade to a newer version?

Reply:
For this PR, I can revert everything that would be require a change to the .form file. If you could tell me what those things are.

Reply:
GUI migration must be done for all forms someday (some forms are fine, some forms must be recreated due miss form file, some forms must be edited due miss form's changes like your use case, some forms must be migrated from IntelliJ format to NetBeans). As I remember -- newer netbeans (I tested v10 long ago) has some compatibility problems with xmage's code base and editors, what's why version 8 required.

You can find a v8 workable downloadable link in the wiki: https://github.com/magefree/mage/wiki/Setting-up-your-Development-Environment#steps-to-retrieve-build-and-run-xmage

Reply:
> I can revert everything that would be require a change to the .form file. If you could tell me what those things are.

Just duplicate same renames and text changes in the `*.form` file too (it's simple xml format).
![shot_220703_015910](https://user-images.githubusercontent.com/8344157/177017302-42927b12-e289-417d-a785-456e86d15c7d.png)
![shot_220703_020118](https://user-images.githubusercontent.com/8344157/177017303-5a3b4fba-e8be-48d3-b03e-c8ca6ee9410a.png)
![shot_220703_020224](https://user-images.githubusercontent.com/8344157/177017316-4e3afcd6-fd13-45d6-8512-72e7f5520d62.png)

Reply:
Okay I think I got them all.